mkdocs.yml

```diff
@@ -2,13 +2,3 @@ docs_dir: ./docs
 site_name: Subscription Watch Documentation
 plugins:
 - techdocs-core
-
-nav:
-- Home: index.md
-- Api: api-documentation.md
-- Iqe EE Proxy: iqe-ee-proxy.md
-- Unleash: unleash.md
-- Kafka topic: kafka-topics.md
-- Swatch Internal Docs: /docs/default/Component/swatch-internal-docs
```

Comment:
The main reason I needed the navigation was to create relative links between the documents. Not sure if there is an alternative way to do this without nav